lib/mutant/integration/minitest.rb

```diff
@@ -0,0 +1,114 @@
+require 'minitest'
+
+# monkey patch for minitest
+module Minitest
+  # prevent autorun from running tests when the vm closes
+  def self.autorun
+  end
+end
+
+module Mutant
+  class Integration
+    # Minitest integration
+    class Minitest < self
+      # represents the data associated with a test
+      class TestCase
+        attr_accessor :class_name
+        attr_accessor :test_method
+
+        def initialize(class_name, test_method)
+          @class_name = class_name
+          @test_method = test_method
+        end
```

Comment:
Mutant depends on [concord](https://github.com/mbj/concord), which allows to combine this:

```
        attr_accessor :class_name
        attr_accessor :test_method

        def initialize(class_name, test_method)
          @class_name = class_name
          @test_method = test_method
        end
```

Into:

```
include Concord::Public.new(:class_name, :test_method)
```

Which is shorter to write/read and requires less unit testing as concord is mutation covered already and the injected semantics are trustwrothy because "trust your dependencies".